### DISCUSSION TITLE
Can't install Cardinal from deb-File

### DISCUSSION THREAD

**Discussion by hr30** [DISTRHO/Cardinal#203] [General]
I downloaded _cardinal_22.03-1kxstudio2_amd64.deb_ from [](https://kx.studio/Repositories:Plugins) and tried to install it, but it fails due to missing dependency _cardinal-data_. This is already included in the deb-file too. I found the LV2 already as so-files,  copied it manually to the LV-directory and started it in Ardour 6.5. But I got error-message 
"/usr/share/cardinal/ does not exist. Make sure that Cardinal was downloaded and installed correctly".
I downloaded the files _cardinal+deps-22.03.tar.xz_ and Cardinal-linux-x86_64-22.03.tar.gz and hoped to get the data from there, which are needed for that directory, but without success.
FYI: I use Linux Debian 11.2 distribution.
Not sure, if this is an issue or something only to be discussed, so I put this into _Discussions_ - hope this is the correct way. 
Thanks in advance for your support.

**Comment by falkTX** [comment]
Please read the repository instructions at the top.
At https://kx.studio/Repositories:Plugins it is clearly stated:

> If a "data" download is listed, it needs to be installed first before the architecture-dependent one. 

So just do that, and it will work for the deb files.

**Reply by hr30** [reply]
Mea Culpa! I've overlooked the data-link. I'm getting old :-(
Anyway: 
Learned how to proceed with data files
Now everything's clear and I look forward to explore the synth.

Thanks for the hint, falkTX

**Comment by joshgura** [comment]
downloading the 500mb archive from Artefacts on Github doesn't seem to contain a standalone executable that will run. at least on linux. the latest package hasn't been updated yet for Manjaro at this particular moment otherwise using pacam works, and it  installs lots of folders and files outside of my /home directory which I do not like, but many programs seem to do that.  The VST3 seems to be self contained however, not dropping all kinds of files and folders in the / part of my operating system. Why can't the standalone be more like that?

**Reply by falkTX** [reply]
I dont understand what you are trying to say.
The standalone extracts the needed resource to run in /tmp
You can use the LV2 or VST plugins instead of the standalone.

**Comment by joshgura** [comment]
After extraction, when I run the standalone nothing happens.
